src/search.cpp

```diff
@@ -1250,6 +1250,11 @@ namespace {
       {
           Depth r = reduction<PvNode>(improving, depth, moveCount);
 
+#ifdef ANTI
+          if (pos.is_anti() && pos.can_capture())
+              r -= r ? ONE_PLY : DEPTH_ZERO;
+          else
```

Comment:
> I'll start running tests on my PC with each individual opening position, and see if I can also generate a corpus of middlegame positions (to test each individually) in order to identify whether this patch is a uniform Elo gain uniformly scaling across all classes of positions

I am not sure whether you are serious. I find it strange either way.

> there is also evidence in FICS computer rating lists that proof-number search is an excellent approach

Do you have a reference for the rating list? You have mentioned before that proof-number search is a good approach and I agree, but despite that I kept improving giveaway SF and it looks like it performs well. E.g., on FICS @lantonov's SF account shows excellent results in suicide chess (http://ficsgames.org/cgi-bin/search.cgi?Games=dsearch;statsdays=365;white=FishTest;variant=13;fyear=2014;tyear=2017;colors=1) and, although he uses an opening book as far as I know, the results with black are telling.

Anyway, I don't know why the existence of other strong engines speaks against trying to improve SF. Also, antichess is weakly solved, but a strong engines can not only be used for positions in the proof tree, which is great for analysis purposes and also, e.g., if you want to study antichess960, because I doubt that there will be 959 more proof trees in the next years.